### ISSUE TITLE
Handling of Vertical Profiles as timeseries in EDR / covJSON responses

### ISSUE THREAD

**Issue by juerg1s** [EURODEO/rodeo-edr-profile#24]
Responses for timeseries as Vertical Profiles is somehow unclear - CovJSON responses has domains for timeseries and vertical profiles, but not for timeseries at different heights. 

Example 
https://interpol-dev.met.no/collections/meps-det-vertical/instances

**Comment by ernstdevreede** [comment]
How about returning a CoverageCollection of vertical profile coverages for the different time steps?

**Comment by mrauhala** [comment]
I agree with Ernst, that is the most logical way to do it. And that is how our current implementation actually works.

Example: https://opendata.fmi.fi/edr/collections/ecmwf_painepinta/instances
You can do /position query with time range and get CoverageCollection of VerticalProfile coverages.
